docs/csaf_checker.md

```diff
@@ -3,7 +3,7 @@
 ### Usage
 
 ```
-csaf_checker [OPTIONS]
+csaf_checker [OPTIONS] domain...
```

Comment:
Is the `...` the common notation for "you can have multiple domains"?

Reply:
I think so ... `LANG=C cat --help | head -1` gives:
```
Usage: cat [OPTION]... [FILE]...
```

Maybe  we should use `[OPTION]... domain...` ?

Reply:
Tough question. It looks like the go default is just `[OPTIONS]`. But I like to Linux version as it is

Reply:
BTW: Whatever we decide - I'm going to implement it...

Reply:
Usually it is good to follow the 
 * https://www.gnu.org/prep/standards/html_node/Command_002dLine-Interfaces.html  GNU Coding standards
which point to the POSIX standards, they extend.
 * https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/
 
Looking there and at a few examples, it seems that several things are implemented and possible,
both with
```
  utility_name [options][operands]
```
and with dots like `...` 
```
chgrp -R [-H|-L|-P] group file...
```
and
```
utility_name [-g option_argument]...[operand...]
```

As the checker has many options, ```[options]``` or ```[OPTIONS]``` seems best practice
as we cannot list all options in there and the dots are used to indicate that one options can be used
several times (in posix). As we can name the operand specifically, either `domain` or a better term
we should list them and then use the dots, so I propose to use

```
csaf_checker [OPTIONS] domain...
```
(it is shorter and seems closer to posix.)